app/js/views/home.js

```diff
@@ -2,15 +2,35 @@ var app = app || {};
 
 app.views = app.views || {};
 
+
 app.views.Home = (function() {
 
 	'use strict';
 
 	return Backbone.View.extend({
 
+		collection: app.trees,
 		className: 'home',
 		template: '#template-home',
 
+		getTrees: function(cb) {
+
+			var self = this;
+			var treesArray = [];
+			this.collection.fetch({
+				success: function(response) {
+					_.each(response.toJSON(), function(item) {
+						console.log('Successfully got tree with id: ' + item.id);
+					})
+					treesArray = response.toJSON();
+					cb(treesArray);
```

Comment:
When calling a callback function in the case of success, the first argument is `null` because there is no error. So the signature for the `cb` argument is `cb(error, trees)`. This is the generally accepted pattern asynchronous functions with callbacks.

In this case, the callback  @should be called as `cb(null, treesArray)`